app/services/reports/region_cache_warmer.rb

```diff
@@ -4,26 +4,72 @@ def self.call
       new.call
     end
 
+    def self.create_slack_notifier
+      return if ENV["SIMPLE_SERVER_DEPLOYMENT_NOTIFICATIONS_WEBHOOK"].blank?
```

Comment:
We wanna use a new webhook but adding env vars to prod is painful right now, so reusing what we have for now.